ichnaea/locate/provider.py

```diff
@@ -611,16 +614,34 @@ def limit_reached(self):
             self.redis_client,
             self.get_ratelimit_key(),
             maxreq=self.ratelimit,
-            expire=60,
+            expire=self.RATE_LIMIT_EXPIRE,
             fail_on_error=True,
         )
 
-    def locate(self, data):
-        location = self.location_type(query_data=False)
+    def _get_cache_key(self, cell_data):
+        return 'fallback_cache_{}'.format(sorted(cell_data.items()))
```

Comment:
That's Python 2.7 syntax, in 2.6 there needs to be a number or a name inside the curly braces.